tsdb/chunkenc/chunk_test.go

```diff
@@ -110,24 +110,23 @@ func testChunk(t *testing.T, c Chunk) {
 }
 
 func benchmarkIterator(b *testing.B, newChunk func() Chunk) {
+	const samplesPerChunk = 250
 	var (
 		t   = int64(1234123324)
 		v   = 1243535.123
 		exp []pair
 	)
-	for i := 0; i < b.N; i++ {
+	for i := 0; i < samplesPerChunk; i++ {
```

Comment:
A non-blocking comment for this PR: we could actually merge this loop with chunk creation loop below. `exp` is not required at all.